apps/rush-lib/src/logic/NodeJsCompatibility.ts

```diff
@@ -32,51 +41,67 @@ export interface IWarnAboutCompatibilityIssuesOptions extends IWarnAboutVersionT
  * @internal
  */
 export class NodeJsCompatibility {
-  public static warnAboutCompatibilityIssues(options: IWarnAboutCompatibilityIssuesOptions): boolean {
-    // Only show the first warning
-    return (
-      NodeJsCompatibility.warnAboutVersionTooOld() ||
-      NodeJsCompatibility.warnAboutVersionTooNew(options) ||
-      NodeJsCompatibility.warnAboutOddNumberedVersion() ||
-      NodeJsCompatibility.warnAboutNonLtsVersion(options.rushConfiguration)
-    );
-  }
-
-  public static warnAboutVersionTooOld(): boolean {
-    if (semver.satisfies(nodeVersion, '< 10.13.0')) {
-      // We are on an ancient version of Node.js that is known not to work with Rush
+  /**
+   * This reports if the Node.js version is known to have serious incompatibilities.  In that situation, the user
+   * should downgrade Rush to an older release that supported their Node.js version.
+   */
+  public static reportAncientIncompatibleVersion(): boolean {
+    // IMPORTANT: If this test fails, the Rush CLI front-end process will terminate with an error.
+    // Only increment it when our code base is known to use newer features (e.g. "async"/"await") that
+    // have no hope of working with older Node.js.
+    if (semver.satisfies(nodeVersion, '< 8.9.0')) {
       console.error(
         colors.red(
           `Your version of Node.js (${nodeVersion}) is very old and incompatible with Rush. ` +
-            `Please upgrade to the latest Long-Term Support (LTS) version.`
+            `Please upgrade to the latest Long-Term Support (LTS) version.\n`
```

Comment:
It doesn't have to strictly be the latest LTS version. May be better to explicitly list the major versions this release of Rush supports.

Reply:
They can probably figure that part out.  I am thinking about adding a page on the website  that explains Rush's support for different versions.